android/app/build.gradle

```diff
@@ -162,6 +162,9 @@ android {
 
         //Fix problem in bt where .env.bt is not recognized
         resValue "string", "build_config_package", "org.pathcheck.covidsafepaths"
+
+        // Handle EN Express deep links
+        manifestPlaceholders = [enxDomain: project.env.get("ENX_APPLINKS_DOMAIN") ?: "*.en.express"]
```

Comment:
any issues with this wildcard being the default? i.e if an app doesnt need deeplinks

Reply:
It will show a chooser dialog and the user can select to open the link in our app or Google Chrome (for example).
Since it cannot verify the app is linked to the website it won't be opened automatically.

That's the only thing that will happen when using a wildcard